src/_includes/components/case-study-category-card.njk

```diff
@@ -3,7 +3,7 @@ We should only need a selection of 6 assets as we only show 6 case study categor
 
 <div class="bg-grey-300 border-grey-500 border-px border-solid relative rounded-2">
   <div class="aspect-ratio aspect-ratio--16x9">
-    <img alt="" class="absolute left-0 rounded-t-2 top-0" loading="lazy" src="/assets/bitmaps/photo-water-02.jpg" />
+    <img alt="" class="absolute h-full left-0 rounded-t-2 top-0 w-full" loading="lazy" src="/assets/bitmaps/photo-water-02.jpg" />
```

Comment:
Question: Do we want these to be `height: auto`, should the images being uploaded not always be cut to perfect 16:9 ratio (as might happen over time)? Or do we want to use `h-full` and also ensure it has `object-fit: cover`?

Reply:
@adamduncan In the context of a `card` I think we should go with `h-full` and `w-full` and probably use `object-fit: cover` otherwise we can end up in a situation where the image won't fill all the space requiring us to use a bg color.

E.g. This is actually the current implementation for the `articleCard` which uses `object-fit: contain` so it doesn't crop any of the image.

<img width="320" alt="Screenshot 2021-06-03 at 15 37 45" src="https://user-images.githubusercontent.com/13508865/120663113-a996f500-c481-11eb-9951-5f88d0732f4a.png">

I agree that it's very likely over time the images will not be a perfect 16:9 ratio but when it comes to `cards` we need to be consistent with the layout so using `h-full`, `w-full` and `object-fit: cover` will be the best approach.

If there's no objections I'll go back through the code and update to `aspect-ratio--cover` for all the cards.

Reply:
Yep, sounds great mate. Thanks 👍